### PR TITLE
[Improment]add be avail cpu num metric

### PR DESCRIPTION
**Pull request by wangbo**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note
1 Add doris_be_avail_cpu_num metric.
2 make be conf immutable

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [x] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

File: be/test/util/cgroup_util_test.cpp [hunk 1]
```diff
@@ -87,4 +87,35 @@ TEST_F(CGroupUtilTest, memlimit) {
...
+
+    std::string cpuset_3_path(dir_path + "/util/test_data/cgroup_cpu_data/cpuset3");
+    std::string cpuset_3_str = "";
+    Status ret3 = CGroupUtil::read_string_line_from_cgroup_file(cpuset_3_path, &cpuset_3_str);
+    EXPECT_TRUE(ret3.ok());
+    int cpu_count3 = 0;
+    static_cast<void>(CGroupUtil::parse_cpuset_line(cpuset_3_str, &cpu_count3));
+    EXPECT_TRUE(cpu_count3 == 10);
+
+    int ret = CGroupUtil::get_cgroup_limited_cpu_number(16);
```
Comment (yiguolei) [inline be/test/util/cgroup_util_test.cpp:117]: 把这个get_cgroup_limited_cpu_number 方法里的函数拆分一下，分别然后我们提供一个cpu.quota 或者cpu.max 这些文件，把cgroup cpu 相关文件的解析，都通过一些我们提供的测试文件都测试到。
不要依赖当前系统的，比如你这里写的>0，我感觉没啥用。

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**